distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/JvmOption.java

```diff
@@ -124,6 +128,13 @@ private static List<String> flagsFinal(final List<String> userDefinedJvmOptions)
         }
     }
 
+    @SuppressForbidden(reason = "ProcessBuilder takes File")
+    private static void setWorkingDir(ProcessBuilder builder) throws IOException {
+        // set temp dir as working dir so it is writeable
```

Comment:
We do we need this? Is this temporary, before we switch to `logs`? If so I think it deserves an extended comment

Reply:
We don't yet know the logs directory when we are determining final flags. And we don't need to, we just need a directory that is writeable, so this uses a temp directory for this jvm invocation. I'll add a comment.